maven-scm-test/src/main/java/org/apache/maven/scm/ScmTckTestCase.java

```diff
@@ -98,14 +101,14 @@ public abstract void initRepo()
     /**
      * {@inheritDoc}
      */
-    protected void setUp()
+    public void setUp()
```

Comment:
Apparently, this still works (at least, no tests seem to be failing because this doesn't run) but wouldn't it be more nice to annotate with `@Before` (in JUnit 4 style)?

Reply:
fixed